ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1157,6 +1161,42 @@ def test_ds_configcheck_passwordstorage(self, modify_pwdstoragescheme):
                 assert "cn=config" in check["kw"]["items"]
                 assert error_msg in check["kw"]["msg"]
 
+    @pytest.fixture
+    def create_logfile(self):
+        """
+        This fixture creates an empty healthcheck.log file when
+        it is not present in /var/log/ipa/healthcheck/ directory so that
+        sosreport command collects the log file in the report generated
+        by the dry run and then deletes the file once the test is finished.
+        """
+        if not self.master.transport.file_exists(HEALTHCHECK_LOG):
+            self.master.run_command(["touch", HEALTHCHECK_LOG])
+        yield
+        self.master.run_command(["rm", HEALTHCHECK_LOG])
+
+    def test_sosreport_includes_healthcheck(self, create_logfile):
+        """
+        This testcase checks that sosreport command
+        when run on IPA system with healthcheck installed
+        collects healthcheck.log file
+        """
+        caseid = "123456"
+        msg = "[plugin:ipa] collecting path '{}'".format(HEALTHCHECK_LOG)
+        cmd = self.master.run_command(
+            [
+                "sosreport",
+                "-o",
+                "ipa",
+                "--case-id",
+                caseid,
+                "--batch",
+                "-v",
+                "--build",
+            ]
+        )
+        assert cmd.returncode == 0
```

Comment:
This can be removed as `raiseonerr=True` is set by default to `run_command()`

Reply:
@mrizwan93 thank you for the review. Have removed the line